### DISCUSSION TITLE
Background job to remove stale sessions from databases with optional expiry time

### DISCUSSION THREAD

**Discussion by beckend** [maxcountryman/tower-sessions#17] [Q&A] [answered]
Is there such a feature?
I can imagine the data in db after a while is going to be massive.

**Comment by maxcountryman** [comment, marked as answer]
Hi there, yes in fact there is. For example, each of the SQLx and MongoDB stores have [a method](https://docs.rs/tower-sessions/0.1.0/tower_sessions/struct.SqliteStore.html#method.continuously_delete_expired) that's intended to be run as a task.

**Comment by beckend** [comment]
Wow thanks, this should be on the main page.

**Reply by maxcountryman** [reply]
Happy to accept PRs that improve the docs. 🙂

**Reply by beckend** [reply]
Is is done.
